src/main/resources/assets/securitycraft/lang/es_es.json

```diff
@@ -0,0 +1,768 @@
+	"messages.securitycraft:universalOwnerChanger.noBlockClaiming": "No puedes reclamar bloques sin propietario.",
+	"messages.securitycraft:irc.connected": "El chat de IRC no es compatible desde la v1.8.3 de SecurityCraft. ¡Si necesitas ayuda, dirígete al canal «#help» de SecurityCraft en Discord!",
+	"messages.securitycraft:cageTrap.captured": "#player ha sido atrapado en #location.",
+	"messages.securitycraft:frame.rightclick": "Haz clic derecho con un monitor vinculado para ver.",
+	"messages.securitycraft:command.sc.usage": "Usa: «/sc connect», «/sc disconnect», «/sc contact» O «/sc resume».",
+	"messages.securitycraft:naming.alreadyMatches": "Este nombre ya coincide con «#n».",
+	"messages.securitycraft:naming.error": "No puedes poner al bloque el nombre «#n», contiene el caracteres no válidos (#c).",
+	"messages.securitycraft:thanks": "¡Gracias por usar SecurityCraft, #!",
+	"messages.securitycraft:tip": "Consejo:",
+	"messages.securitycraft:tip.scHelp": "Escribe «/sc help» para ver el manual de SecurityCraft, que te mostrará ayudas sobre los bloques y objetos de SecurityCraft.",
+	"messages.securitycraft:tip.trello": "Echa un vistazo al tablón de Trello de SecurityCraft; ¡informa de errores o haznos sugerencias!",
+	"messages.securitycraft:tip.patreon": "¿Quieres apoyar a SecurityCraft?",
+	"messages.securitycraft:tip.discord": "¡Únete a nuestro Discord para chatear y recibir notificaciones de actualizaciones y anuncios de SecurityCraft!",
+	"messages.securitycraft:tip.scserver": "¡El servidor oficial de SecurityCraft ha vuelto! Únete en la IP: 64.38.249.237",
+	"messages.securitycraft:universalKeyChanger.notKeypad": "Haz clic derecho en un teclado con el cambiador de contraseñas universal para modificarla.",
+	"messages.securitycraft:universalKeyChanger.passcodeChanged": "Contraseña cambiada con éxito.",
+	"messages.securitycraft:passwordProtected.notSetUp": "Este bloque no tiene contraseña aún. No puedes interactuar con él.",
+	"messages.securitycraft:sentry.needsBlockBelow": "¡Tienes que colocar la torreta en lo alto de un bloque!",
+	"messages.securitycraft:sentry.needsBlocksAround": "¡Asegúrate de que tu torreta está rodeanda de bloques por los cuatro lados!",
+	"messages.securitycraft:sentry.mode1": "Modo: Agresiva (siempre activa, ataca a criaturas hostiles y jugadores).",
+	"messages.securitycraft:sentry.mode2": "Modo: Camuflada (se activa cuando un jugador se acerca y le ataca).",
+	"messages.securitycraft:sentry.mode3": "Modo: Apagada (no se activa, no ataca).",
+	"messages.securitycraft:blockpocket.multipleManagers": "No hay más de un gestor de bloque secreto en la estructura.",
+	"messages.securitycraft:blockpocket.invalidBlock": "¡Bloque «%2$s» no válido en %1$s!",
+	"messages.securitycraft:blockpocket.invalidBlock.pillar": "¡Bloque «%2$s» no válido en %1$s! (¡asegúrate de que el pilar esté rotado correctamente!)",
+	"messages.securitycraft:blockpocket.unowned": "¡No eres propietario del bloque «%2$s» en %1$s!",
```

Comment:
«%2$s» -> \"%2$s\"

Why changing all this?

Reply:
The variable is exactly the same. The " is changed to « » because it is changed in the game translation itself too.

Reply:
You mean MC renders « » in Spanish while it renders "" in English?

Reply:
I meant that in the lang file it was translated as « » instead of " " in 1.15 and 1.16. But I have forced the action to see the text and it is automatically changed to " ", so I will change it to the English quotes.

Reply:
I thought it would appear properly in-game, in 1.15 and 1.16 they were used in the game lang file, but finally they appear as English quotes ingame.